docs/src/references/c8y-configuration-management.md

```diff
@@ -192,6 +193,10 @@ Note that the `c8y_configuration_plugin` does not create any child-device twin i
 
 For aspect (2) there are two proposals as below. Decision has to been taken which proposal to follow.
 
+TODO: Add some short summary what those proposal shall achieve / there scope.
+
+--------------------------------------------------------------------------------
+
 **Proposal 1:**
```

Comment:
Having proper section titles would be better. Both markdown and wording can be improved.

```suggestion
### Deliver and update configuration files over MQTT
```

Reply:
Improved now, see https://github.com/cstoidner/thin-edge.io/commit/0e2f15737d6652ba84aebf475bf710d3d560099f